PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRecursiveSoftDrop.h

```diff
@@ -83,8 +83,8 @@ class AliAnalysisTaskRecursiveSoftDrop : public AliAnalysisTaskEmcalJet {
 
   
   Int_t                               fContainer;              // jets to be analyzed 0 for Base, 1 for subtracted. 
-  Double_t                            fShapesVar_Det[4];       // jet shapes used for the tagging
-  Double_t                            fShapesVar_True[4];      // jet shapes used for the tagging
+  Double_t                            fShapesVar_Det[5];       // jet shapes used for the tagging

```

Comment:
You should update your ClassDef, and you should really be setting whether members are streaming or not (but perhaps you can get away with this until after QM)